.github/workflows/label-actions.yml

```diff
@@ -0,0 +1,22 @@
+name: "Label Actions"
+
+on:
+  issues:
+    types: [labeled, unlabeled]
+  pull_request_target:
+    types: [labeled, unlabeled]
+  discussion:
+    types: [labeled, unlabeled]
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+  discussions: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      # TODO_TECHDEBT: switch to `dessant/label-actions@v3` when https://github.com/dessant/label-actions/pull/29 is merged.
+      - uses: okdas/label-actions@patch-1
```

Comment:
What kind of actions do you want this workflow to perform? It requires another file: `.github/label-actions.yml` - https://github.com/pokt-network/poktroll/blob/main/.github/label-actions.yml - which contains the different actions that can happen when the label is present.

Reply:
Noted. This is good insight. My goal was to get `In Review` to automatically switch to `Done` once merge in the PR. 

I couldn't find a GitHub setting for it, so I assume this is the  tool to make that happen.

Do you know if that's a default elsewhere?

Reply:
@Olshansk  I was actually curious how this works and this is what I found: https://github.com/orgs/pokt-network/projects/144/workflows

Apparently this is a new feature github released earlier this year: https://github.blog/changelog/2024-04-25-github-issues-projects-auto-close-issue-project-workflow/

Reply:
Interesting!

I just added a few of them to our grove project as well: https://github.com/orgs/buildwithgrove/projects/1/workflows/32463471